src/Elliptic/DiscontinuousGalerkin/DgElementArray.hpp

```diff
@@ -53,49 +44,48 @@ struct DgElementArray {
   using const_global_cache_tag_list =
       Parallel::get_const_global_cache_tags_from_pdal<PhaseDepActionList>;
 
-  using add_options_to_databox = AddOptionsToDataBox;
+  using allocation_tags =
+      tmpl::list<::Tags::Domain<volume_dim, Frame::Inertial>,
+                 ::Tags::InitialRefinementLevels<volume_dim>>;
 
-  using options = tmpl::list<typename Metavariables::domain_creator_tag>;
+  using initialization_tags = Parallel::get_initialization_tags<
+      Parallel::get_initialization_actions_list<phase_dependent_action_list>,
+      allocation_tags>;
```

Comment:
I think it can be done, but it is not as simple a change as I had hoped, so I would rather make it in a subsequent PR as I would rather get this in and make the change for the tags in the ConstGlobalCache so I'm not blocking other PRs

Reply:
Doing this in another PR seems fine, perhaps open an issue then.